spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/command/ConfigCommands.java

```diff
@@ -257,8 +257,13 @@ public String target(
 		SecurityInfo securityInfo = about.getSecurityInfo();
 		modelBuilder.addRow().addValue("Security").addValue(securityInfo);
 		rowIndex++;
-		if (securityInfo.isAuthenticated()) {
-			modelBuilder.addRow().addValue("Roles").addValue(securityInfo.getRoles());
+
+		if (securityInfo.isAuthenticated() && securityInfo.isAuthorizationEnabled()) {
```

Comment:
I think it should be: https://gist.github.com/trisberg/cd5e9f1930b9446541d86d0ac57e7092